lib/restful_resource_bugsnag/middleware.rb

```diff
@@ -47,5 +47,11 @@ def call(notification)
     def request_host_from_exception(exception)
       URI.parse(exception.request.url).host
     end
+
+    def attempt_json_parse(string)
+      JSON.parse(string)
+    rescue JSON::ParserError
+      string
+    end
```

Comment:
I did this so we don't crash if we ever get a non JSON response.

Reply:
yeah makes sense

Reply:
I think the body can be `nil`*, so we should probably either `to_s` it or rescue `TypeError`s also

* going on this https://app.bugsnag.com/carwow/quotes-site/errors/5e62650c881e1100177205f2?filters[event.since][0]=30d&filters[error.status][0]=snoozed

Reply:
well spotted @revdan , I'll fix

Reply:
done @revdan